### ISSUE TITLE
No formatting (bold, italic, etc.) preserved?

### ISSUE THREAD

**Issue by DexyZ** [fabianonline/telegram_backup#102]
I was going to write about it in #15 or #101, but it seems to me that the issue is with SQLite rather than HTML. The database stores plain text messages and does not preserve markup for `**bold**` or  `__italic__`, does it?

**Comment by fabianonline** [comment]
Yes, it does. But it also saves the raw message object in the column `data`(or whatever name I gave it, I'm too lazy to look it up right now). So it's not gone. I "just" have to add code to the exporter to display these styles as well...
